conbench/entities/_entity.py

```diff
@@ -139,7 +139,7 @@ def one(cls, **kwargs):
 
     @classmethod
     def first(cls, **kwargs):
-        return current_session.scalars(select(cls).filter_by(**kwargs)).first()
+        return current_session.scalars(select(cls).filter_by(**kwargs).limit(1)).first()
```

Comment:
This change got me curious.

Naive assumption was that `first()` does some LIMIT under the hood.

in the legacy query API this was documented:

> first() applies a limit of one within the generated SQL

But seemingly that was removed; from the migration guide (that's probably where you have it from?):

![image](https://github.com/conbench/conbench/assets/265630/bc137066-165b-4c8a-9ff9-b88052ea947a)

Reply:
Yes, isn't this wild? I went down the same rabbit hole and should have left a comment.